### PR TITLE
Ghosts can follow mechs

### PR DESCRIPTION
**Pull request by Crazylemon64**
:cl:Crazylemon
rscadd: Ghosts can now follow mecha
/:cl:

### CODE REVIEW

File: code/_onclick/observer.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 			return									// seems legit.
 
 	// Things you might plausibly want to follow
-	if((ismob(A) && A != src) || istype(A,/obj/machinery/bot) || istype(A,/obj/singularity))
+	if((ismob(A) && A != src) || istype(A,/obj/machinery/bot) || istype(A,/obj/singularity) || istype(A,/obj/mecha))
```
Comment (Iamgoofball) [inline code/_onclick/observer.dm]: holy shit this is horrible FUCK

remove this entire check

port ghost code from /tg/ where we can orbit any object/mob and has a Point of Interest system so you can add stuff to be followable on the fly

Comment (Spacemanspark) [inline code/_onclick/observer.dm]: Orbiting as a ghost on Paradise would be hilarious, considering it shows what we looked like when we died/ our selected character in the character setup option should you observe.

Comment (taukausanake) [inline code/_onclick/observer.dm]: Let's just have both, normal follow and orbiting. Bonus if one day we can have humans see ghosts and have them all congregate into a single shell haunting

Comment (Crazylemon64) [inline code/_onclick/observer.dm]: yeah this is crummy, but I have real life stuff right now

**Pull request merged**